src/React.Owin/JsxFileMiddleware.cs

```diff
@@ -0,0 +1,71 @@
+﻿/*
+ *  Copyright (c) 2014-2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant 
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Microsoft.Owin.StaticFiles;
+
+namespace React.Owin
+{
+	/// <summary>
+	/// Enables serving static JSX files transformed to pure JavaScript. Wraps around StaticFileMiddleware.
+	/// </summary>
+	public class JsxFileMiddleware
+	{
+		private readonly StaticFileMiddleware _internalStaticMiddleware;
+
+		static JsxFileMiddleware()
+		{
+			Initializer.Initialize(_ => _);
```

Comment:
This should be passed a per-request lifetime so the correct cleanup is done at the end of the request. I think at the moment nothing is being disposed properly at the end of the request (or all the per-request objects are actually singletons). In ASP.NET I use `HttpContext.Current` for that (see https://github.com/reactjs/React.NET/blob/master/src/React.Web/WebInitializer.cs and https://github.com/reactjs/React.NET/blob/master/src/React.Web/TinyIoCAspNetExtensions.cs). I'm not experienced with OWIN so I'm not sure how to do this properly. Does OWIN have its own IoC container? How do other IoC containers integrate with it?